src/ThreeEditor/ThreeEditor.tsx

```diff
@@ -31,8 +31,12 @@ function ThreeEditor(props: ThreeEditorProps) {
 	}, [editor, props.onEditorInitialized]);
 
 	useEffect(() => {
-		props.focus === true && containerEl.current?.focus();
+		if (props.focus === true) {
+			containerEl.current?.focus();
+			editor?.signals.sceneGraphChanged.dispatch();
+		}
 		props.focus === false && containerEl.current?.blur();
```

Comment:
what if the focus is undefined? why focus can be undefined?

Reply:
Because it is an optional property of the component to indicate if ThreeEditor is focused, but is not always necessary.

Reply:
Now is value required